### DISCUSSION TITLE
Особенности в отображении мобильной версии

### DISCUSSION THREAD

**Discussion by tvideo** [totumonline/totum-ru-issues-and-discussions#129] [Неопределенное]
Здравствуйте. 
Подскажите, плз, со следующим вопросиком:
при вводе данных любых данных (префильтр, либо строчная часть), с мобильного телефона, при смене ориентации экрана, сбрасывается ввод.

https://user-images.githubusercontent.com/32614108/168010317-4c25f50d-19eb-46f3-b509-c937d2951d58.mp4

В чем может быть причина? 
Спасибо

**Comment by ttm-alexeypolunin** [comment]
Есть сложная логика открытия-закрытия полей на редактирование. Связана она с тем, что бы исключить зомби-поля — открытые на редактирование, но которые нельзя закрыть.

Есть сложная логика ресайза окна при повороте — на андроидах и айфонах ресайз работает по разному и есть стечение обстоятельств уводящее в бесконечную перезагрузку страницы.

Поэтому при повороте все поля открытые на редактирование закрываются и требуется подтверждение на перерисовку.

Возможно можно будет сделать лучше с нативной версией или новым фронтом, но это не ранее $1.5M инвестиционных денег :)

**Reply by tvideo** [reply]
Понял, принял. Спасибо
